NOTICE.txt

```diff
@@ -216,3 +216,13 @@ In addition, certain files include the notices provided below.
 // shall not be used in advertising or otherwise to promote the sale,
 // use or other dealings in these Data Files or Software without prior
 // written authorization of the copyright holder.
+
+----------------------
+
+/*  Written in 2018 by David Blackman and Sebastiano Vigna (vigna@acm.org)
+
+To the extent possible under law, the author has dedicated all copyright
+and related and neighboring rights to this software to the public domain
+worldwide. This software is distributed without any warranty.
+
+See <http://creativecommons.org/publicdomain/zero/1.0/>.*/

```

Comment:
Not worth resetting testing, but if we need to touch this for some other reason:
```suggestion
See <http://creativecommons.org/publicdomain/zero/1.0/>. */
```